### DISCUSSION TITLE
please reissue license

### DISCUSSION THREAD

**Discussion by YYN85** [aclap-dev/video-downloadhelper#326] [License & Purchase requests]
### What's your issue?

I payed for the license but didn't get the code to my email

Reissue please

### The 10 first digits (only 10) of your license number or email address

444507@mail.ru

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by Northshadow** [comment]
You can reclaim your key here:

https://www.downloadhelper.net/recover-license

**Reply by YYN85** [reply]
My email is not found. If I provide payment conformation, is it possible to solve this problem?

**Reply by jcfrog** [reply]
@YYN85 please send your purchase info to licenses@downloadhelper.net

**Comment by jcfrog** [comment]
@YYN85 hi, sorry it seems you had a monthly subscription that ended in December 2020
